deployment/terraform/metrics.go

```diff
@@ -157,6 +159,31 @@ func (t *Terraform) setupMetrics(extAgent *ssh.ExtAgent) error {
 		}
 	}
 
+	if t.output.HasRedis() {
+		redisEndpoint := fmt.Sprintf("redis://%s", net.JoinHostPort(t.output.RedisServer.Address, strconv.Itoa(t.output.RedisServer.Port)))
+		redisTargets = append(redisTargets, "metrics:9121")
```

Comment:
Nit:
This is the port of the redis exporter, right? I know the rest of definitions here are similar to this one, but they seem a bit cryptic if one doesn't know what they're defining, so maybe we can start slightly improving things  with something like:
```suggestion
		redisExporterPort := 9121
		redisTargets = append(redisTargets, "metrics:"+redisExporterPort)
```

Reply:
Isn't the same in the elasticsearch exporter as well?

Reply:
Most probably, yes, but I noticed it here because it wasn't me the one writing the code :joy: I'm 1/5 here, though,just a  nit :)